examples/button_leds.rs

```diff
@@ -0,0 +1,30 @@
+#![feature(alloc)]
```

Comment:
This shouldn't be required here. Only if you access the `alloc` crate which provides heap-based collections.